gtsfm/utils/ellipsoid.py

```diff
@@ -78,8 +83,9 @@ def remove_outlier_points(point_cloud: np.ndarray) -> np.ndarray:
 
     mags = np.linalg.norm(point_cloud, axis=1)
     cutoff_mag = np.percentile(mags, OUTLIER_DISTANCE_PERCENTILE)
+    inlier_mask = mags < cutoff_mag
```

Comment:
maybe we can use the inlier mask on the line below, also?

Reply:
yup. forgot to change to update that.